### PR TITLE
CKV_AWS_114

### PR DESCRIPTION
**Pull request by avivgu**
By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.

### CODE REVIEW

File: checkov/terraform/checks/resource/aws/EMRClusterKerberosAttributes.py [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+        name = "Ensure that EMR clusters have Kerberos Enabled"
+        id = "CKV_AWS_114"
+        supported_resources = ['aws_emr_cluster']
+        categories = [CheckCategories.GENERAL_SECURITY]
+        super().__init__(name=name, id=id, categories=categories, supported_resources=supported_resources)
+
+    def scan_resource_conf(self, conf):
+        if 'kerberos_attributes' in conf:
+            kerberos_attributes = conf['kerberos_attributes'][0]
+            if hasattr(kerberos_attributes, '__len__') and 'realm' in kerberos_attributes:
```
Comment (rotemavni) [inline checkov/terraform/checks/resource/aws/EMRClusterKerberosAttributes.py:16]: @avivgu Did you mean to check if `len(kerberos_attributes) > 0`?

Comment (avivgu) [reply checkov/terraform/checks/resource/aws/EMRClusterKerberosAttributes.py:16]: Just wanted to make sure kerberos_attributes is an iterable object

Comment (rotemavni) [reply checkov/terraform/checks/resource/aws/EMRClusterKerberosAttributes.py:16]: Then `len(kerberos_attributes) > 0` will do the trick

Comment (avivgu) [reply checkov/terraform/checks/resource/aws/EMRClusterKerberosAttributes.py]: done

**Pull request merged**